torch/nn/functional.py

```diff
@@ -3718,9 +3701,11 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
     Args:
         input (Tensor): the input tensor
         size (int or Tuple[int] or Tuple[int, int] or Tuple[int, int, int]):
-            output spatial size.
-        scale_factor (float or Tuple[float]): multiplier for spatial size. If `scale_factor` is a tuple,
-            its length has to match `input.dim()`.
+            output spatial size.  If `size` is a tuple, its length has to match
+            spatial dimensions in `input.dim()`.
+        scale_factor (float or Tuple[float, ...]): multiplier for spatial size.
```

Comment:
To match this, we could use `Tuple[int, ...]` for `size` no? That would be easier to read.

Reply:
Let’s resolve the other conversation and then unify both accordingly.